### ISSUE TITLE
Error running make: `gcp` is not recognized on zsh shell

### ISSUE THREAD

**Issue by Samia1117** [emichael/dslabs#58]
**Describe the bug**
The file copier tool `gcp` is not recognized on zsh shell on MacOS 13.2.1 (the problem may be more general, but at least covers this extent). For Darwin OS, the preferred file copier ( $(CP) )on dslabs Makefile is `gcp` instead of `cp`. However, `make clean all` fails when `gcp` is attempted to be used (in line: `$(CP) -r labs handout-files/. $(OTHER_FILES) $@`). **On changing `gcp` to `cp` the problem is fixed.** Of course, changing to `cp` might break other cases for which you probably had `gcp` in the first place, but there should at least be a line to check in the Makefile whether the current shell recognizes `gcp` or not. _Users should be able to run `make` out of the box_ once they have the required/recommended tools set up (Python3, Make, Java 14, IntelliJ)

**Screenshots**
![gcp-command-not-found](https://github.com/emichael/dslabs/assets/48843553/a6aad74b-8029-4835-9e0c-e9f831809938)
![macOS-version-shell-info](https://github.com/emichael/dslabs/assets/48843553/3dad6ecc-f574-460f-a374-665a62ef6749)

**Environment**
 - OS: MacOS Ventura 13.2.1
 - JDK (output of `java --version`):
 java 17.0.4.1 2022-08-18 LTS
Java(TM) SE Runtime Environment (build 17.0.4.1+1-LTS-2)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.4.1+1-LTS-2, mixed mode, sharing)
 - DSLabs githash: 35df0e99e3f76da00631658bb111d2f7e411b492 (master branch)

**Comment by wilcoxjay** [comment]
TL;DR:
```
brew install coreutils
```

----

This is documented in this repo's [README](https://github.com/emichael/dslabs/blob/master/README.md#contributing):
> The main tools for development are the same as the students' dependencies — Java 14 and Python 3. You will also need a few utilities such as wget to build with the provided Makefile; MacOS users will need gtar and gcp provided by the coreutils Homebrew package.

Note that this is not required for students. It's only required if you plan to hack on dslabs itself.

**Comment by Samia1117** [comment]
Oh ok, missed that. I was not trying to 'contribute' (at that time) and consulting only the handout/README. I was trying to run `./run-tests.py --lab 0 --test-num 1 `, but it was failing with the same error (scshot). I would think it's a good idea to add the lines you quoted above to the handout/README (Getting Started section) as well. 

![run-python-lab0-test1](https://github.com/emichael/dslabs/assets/48843553/ca1832bf-8c43-459e-87dc-eae688607ff5)
